docs/site/Model.md

```diff
@@ -257,12 +257,12 @@ to better tailor towards the JSON Schema being produced.
 
 ## Other ORMs
 You might decide to use an alternative ORM/ODM in your LoopBack application.
-Loopback v4 no longer expects you to provide your data in its own custom Model
+LoopBack v4 no longer expects you to provide your data in its own custom Model
```

Comment:
v4 offends me. Not really, but `LoopBack 4` seems to be used more throughout the docs